crates/testing/src/test_runner.rs

```diff
@@ -481,7 +482,7 @@ where
                     self.late_start.insert(
                         node_id,
                         LateStartNode {
-                            network,
+                            network: Some(network),
```

Comment:
I'm a bit confused here -- I would've assumed `skip_late == true` means we want to skip initializing the nodes. Shouldn't we set `network: None` in that case?

(I don't actually know -- genuinely just wondering)